src/main/java/com/gregtechceu/gtceu/config/ConfigHolder.java

```diff
@@ -462,9 +462,10 @@ public static class MachineConfigs {
         public boolean enableMaintenance = true;
         @Configurable
         @Configurable.Comment({
-                "Time in ticks between when Multiblocks can require Maintenance. By default, 48 hours.",
-                "Default: 3456000" })
-        public int maintenanceTime = 3456000;
+                "How often to check for maintenance, rolling a 1/6000 chance every X ticks (before secondary effects like Configurable Maintenance Hatch).",
+                "In default settings, this equates to a 5% chance every hour of a machine running.",
+                "Default: 1000 (ticks)" })
+        public int maintenanceCheckRate = 1000;
```

Comment:
I purposefully renamed this since:
1. It serves a very different purpose mathematically compared to before
2. The old default value of this config would lead to an even less likely chance of maintenance occurring than before with the old math